pkgs/development/python-modules/cdxj-indexer/default.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  warcio,
+  surt,
+  idna,
+  py3amf,
+  multipart,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "cdxj-indexer";
+  version = "1.4.6";
+  pyproject = true;
```

Comment:
The package does not have `pyproject.toml` only `setup.py` exists.

Specify `format = "setuptools";` instead.